src/components/userView/user.scss

```diff
@@ -1,5 +1,5 @@
 .user-view {
   margin: 20pt;
   margin-top: 0 !important;
-  overflow: auto;
+  overflow: hidden;
```

Comment:
what was the issue with the scroll bar? we have been using overflow:auto everywhere